diff --git a/arch/arm/boot/dts/aspeed-g6.dtsi b/arch/arm/boot/dts/aspeed-g6.dtsi
index cc2f8b785917..ac2c084784ef 100644
--- a/arch/arm/boot/dts/aspeed-g6.dtsi
+++ b/arch/arm/boot/dts/aspeed-g6.dtsi
@@ -31,7 +31,6 @@ aliases {
 		i2c15 = &i2c15;
 		serial0 = &uart1;
 		serial1 = &uart2;
-		serial2 = &uart3;
 		serial3 = &uart4;
 		serial4 = &uart5;
 		serial5 = &vuart1;
@@ -740,20 +739,6 @@ uart2: serial@1e78d000 {
 				status = "disabled";
 			};
 
-			uart3: serial@1e78e000 {
-				compatible = "ns16550a";
-				reg = <0x1e78e000 0x20>;
-				reg-shift = <2>;
-				reg-io-width = <4>;
-				interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&syscon ASPEED_CLK_GATE_UART3CLK>;
-				resets = <&lpc_reset 6>;
-				no-loopback-test;
-				pinctrl-names = "default";
-				pinctrl-0 = <&pinctrl_txd3_default &pinctrl_rxd3_default>;
-				status = "disabled";
-			};
-
 			uart4: serial@1e78f000 {
 				compatible = "ns16550a";
 				reg = <0x1e78f000 0x20>;
