Test Patch
Upstream-Status: Inappropriate [musl specific]

--- a/arch/arm/boot/dts/broadcom/bcm2835-rpi-a.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2835-rpi-a.dts
@@ -16,6 +16,47 @@ memory@0 {
 	};
 };
 
+/{
+    gpio-sim {
+        compatible = "gpio-simulator";
+
+        bank0: bank0 {
+            gpio-controller;
+            #gpio-cells = <2>;
+            ngpios = <16>;
+            gpio-sim,label = "dt-bank0";
+            gpio-line-names = "power-chassis-good", "power-chassis-control", "power-ffs-sync-history", "GPIO03", "GPIO04", "GPIO05", "GPIO06", "GPIO07",
+                              "presence-ps1", "presence-ps2", "presence-ps3", "presence-ps4", "GPIO12", "GPIO13", "GPIO14", "GPIO15";
+        };
+
+        bank1: bank1 {
+            gpio-controller;
+            #gpio-cells = <2>;
+            ngpios = <8>;
+            gpio-sim,label = "dt-bank1";
+            gpio-line-names = "LED0", "LED1", "LED2", "LED3", "LED4", "LED5", "LED6", "LED7";
+        };
+    };
+
+    leds {
+        compatible = "gpio-leds";
+
+        led-0 {
+            label = "bmc_heartbeat_amber";
+            gpios = <&bank1 0 GPIO_ACTIVE_HIGH>;
+            linux,default-trigger = "heartbeat";
+        };
+
+        led-1 {
+            label = "identify";
+            gpios = <&bank1 1 GPIO_ACTIVE_LOW>;
+            linux,default-trigger = "heartbeat";
+        };
+
+    };
+
+};
+
 &gpio {
 	/*
 	 * Taken from Raspberry-Pi-Rev-1.0-Model-AB-Schematics.pdf
