Test Patch
Upstream-Status: Inappropriate [musl specific]

--- a/redfish-core/lib/power.hpp
+++ b/redfish-core/lib/power.hpp
@@ -297,9 +297,8 @@ inline void requestRoutesPower(App& app)
                 std::move(valueHandler));
         };
 
-        constexpr std::array<std::string_view, 2> interfaces = {
-            "xyz.openbmc_project.Inventory.Item.Board",
-            "xyz.openbmc_project.Inventory.Item.Chassis"};
+        constexpr std::array<std::string_view, 1> interfaces = {
+            "xyz.openbmc_project.Inventory.Item.PowerSupply"};
 
         dbus::utility::getSubTreePaths("/xyz/openbmc_project/inventory", 0,
                                        interfaces, std::move(chassisHandler));
