Test Patch
Upstream-Status: Inappropriate [musl specific]

--- a/arch/arm/boot/dts/broadcom/bcm2835-rpi-a.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2835-rpi-a.dts
@@ -37,6 +37,17 @@ bank1 {
             gpio-line-names = "LED0", "LED1", "LED2", "LED3", "LED4", "LED5", "LED6", "LED7";
         };
     };
+
+    leds {
+        compatible = "gpio-leds";
+
+        led-0 {
+            label = "bmc_heartbeat_amber";
+            gpios = <&bank1 0 GPIO_ACTIVE_HIGH>;
+            linux,default-trigger = "heartbeat";
+        };
+    };
+
 };
 
 &gpio {
