diff --git a/include/configs/evb_ast2600_spl.h b/include/configs/evb_ast2600_spl.h
index ae3e628d55..901d66a6af 100644
--- a/include/configs/evb_ast2600_spl.h
+++ b/include/configs/evb_ast2600_spl.h
@@ -30,6 +30,7 @@
 #define CONFIG_EXTRA_ENV_SETTINGS	\
 	"loadaddr=" __stringify(CONFIG_SYS_LOAD_ADDR) "\0"	\
 	"bootspi=fdt addr 20100000 && fdt header get fitsize totalsize && cp.b 20100000 ${loadaddr} ${fitsize} && bootm; echo Error loading kernel FIT image\0"	\
+    "boottftp=tftp 0x83000000 fit_linux_init.bin;bootm 0x83000000" \
 	"verify=yes\0"	\
 	""
 
